jobsdb/backup.go

```diff
@@ -137,15 +137,23 @@ func (jd *Handle) uploadDumps(ctx context.Context, dumps map[string]string) erro
 		path := filePath
 		operation := func() error {
 			if err := jd.uploadTableDump(ctx, wrkId, path); err != nil {
-				jd.logger.Errorf("[JobsDB] :: Failed to upload workspaceId %v. Error: %s", wrkId, err.Error())
+				jd.logger.Errorw(
+					"failed backup upload",
+					"workspaceID", wrkId,
+					"error", err.Error(),
```

Comment:
It warehouse we have a bunch of constants to simplify consistency and avoid possible typos: https://github.com/rudderlabs/rudder-server/blob/master/warehouse/logfield/logfield.go

It can be done here as well.

Reply:
sidenote:
maybe this `warehouse/logfield` should be placed outside(as `package constants` perhaps, because stats for example use the same keys in a lot places too), because a lot of constants defined in there could be used in other places too.

I'm adding `jobsdb/internal/constants/constants.go`, but think it would better serve placed outside.

Reply:
Yes, sharing it [is being discussed](https://github.com/rudderlabs/rudder-server/pull/3731) already, just not sure how yet plus some of the constants might be domain specific (e.g. uploadID is WH specific).

Reply:
> some of the constants might be domain specific

true, wouldn't be hard to group them though. Personally I'd like to have them in one place.
Anyway, for now I'll continue with `jobsdb/internal/constants`